src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -713,7 +727,9 @@ def decode(
 
         # retrieve offsets
         if output_offsets:
-            offsets = self._compute_offsets(token_ids, time_precision=time_precision)
+            segments = kwargs.get("segments")
+            offsets = self._compute_offsets(token_ids, time_precision=time_precision, longform_timestamps=segments)
```

Comment:
Why keep everything bar one place named as "segments" - does it have a special meaning wrt Whisper?